.markdownlint.yaml

```diff
@@ -1,2 +1,3 @@
 ---
 MD013: false
+MD033: false #  no-inline-html
```

Comment:
Is the comment here really needed? Searching `MD033` immediately returns the rule. If commenting then `MD013` should have one too

Reply:
🤷 David suggested and it doesn't hurt...?

Reply:
But consistency

Reply:
Added more comments.